cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -194,7 +194,7 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             this.$('input.date').datepicker({'dateFormat': 'm/d/yy'});
             this.$('input.time').timepicker({
                 'timeFormat' : 'H:i',
-                'forceRoundTime': true
+                'forceRoundTime': false
```

Comment:
Search for "forceRoundTime" and remove the override that Solutions added for a different field (which extends BaseDateEditor).